downloads/swan-lake-release-notes/swan-lake-preview4/RELEASE_NOTE.md

```diff
@@ -1,30 +1,490 @@
+
+Along with the scoping, BIR now contains a new variable kind named `SYNTHETIC`, which identifies the variables generated at the de-sugar phase. These are skipped when adding meta-data so that generated variables do not appear in the scope with a debug hit.
 
 #### Standard Library
-- 
 
-#### Dev Tools
-- 
+Re-align organization names of the connectors. 
+
+|Modules|Old Organisation|New Organisation|
+|:---:|:---:|:---:|
+|mysql| ballerina/mysql| ballerinax/mysql|
+|kafka| ballerina/kafka| ballerinax/kafka|
+|rabbitmq| ballerina/rabbitmq| ballerinax/rabbitmq|
+|nats| ballerina/nats| ballerinax/nats|
+
+#### Developer Tools
+
+##### The OpenAPI Tool
+
+###### OpenAPI CLI
+
+The behavior of the OpenAPI CLI command was changed to make it easier to use. Now, you can generate Ballerina code from an OpenAPI contract and vice versa.
+
+###### OpenAPI to Ballerina
+
+ ```bash
+$  ballerina openapi -i <contract>  -o <output path>
+```
+
+- You can give the contract file in the YAML or JSON format.
+- The above command generates both the service and client stubs.
+- The command will give an output of three files containing the service, client and schema.
+
+    **Command output example:**
+
+    ``` bash
+    $ ballerina openapi -i petstore.yaml  
+    The service generation process is complete. The following files were created.
+    -- petstore-service.bal
+    -- petstore-client.bal
+    -- schema.bal
+    ```
+
+- If you are interested in creating only a service, you can use the `mode` option to give the required mode. The `mode` can be a service or a client.
+
+    ```bash
+    $  ballerina openapi -i <contract> --mode (service | client ) -o <output Path>
+	```
+
+- You can specify the output path with the `-o` option.
+
+    ```bash
+    $  ballerina openapi -i <contract> --service-name <service name> -o <output Path>
+	```
+
+- If you are interested in creating a service with resources having a specific tag or an `operationId`, you can do the following.
+
+    ```bash
+    $  ballerina openapi -i <contract> --tags <tags> -o <output Path>
+	$  ballerina openapi -i <contract> --operations <operationIds> -o <output Path>
+	```
+
+###### Ballerina to OpenAPI
+
+```bash
+     $  ballerina openapi -i <ballerina service file>   -o <output path> 
+```
+
+The service name is not mandatory and you can give the service name if you want to document a specific service. Otherwise, it will generate all the services that are included in the Ballerina source file.
+
+
+```bash
+    $  ballerina openapi -i <ballerina service file>  --service <service Name> -o <output path>
+```
+
+##### Improvements related to code actions
+
+###### Azure Functions
+
+Fix for https://github.com/ballerina-platform/module-ballerinax-azure.functions/issues/33
+
+###### AWS Lambda
+
+Domain-specific event types support
```

Comment:
Shouldn't we say "Added" or something for the completeness of the sentence?

Reply:
Done via https://github.com/ballerina-platform/ballerina-dev-website/pull/1311.